Source/Core/DolphinQt2/DolphinQt2.vcxproj

```diff
@@ -48,9 +48,8 @@
     <Link>
       <AdditionalLibraryDirectories>$(ExternalsDir)ffmpeg\lib;$(ExternalsDir)OpenAL\lib;%(AdditionalLibraryDirectories)</AdditionalLibraryDirectories>
       <AdditionalDependencies>avrt.lib;iphlpapi.lib;winmm.lib;setupapi.lib;opengl32.lib;glu32.lib;rpcrt4.lib;comctl32.lib;avcodec.lib;avformat.lib;avutil.lib;swresample.lib;swscale.lib;%(AdditionalDependencies)</AdditionalDependencies>
-      <SubSystem Condition="'$(Configuration)|$(Platform)'=='Release|x64'">Windows</SubSystem>
```

Comment:
To expand on this, `Windows` is the default for DolphinWX because it follows the rules relating to exported symbol names: https://docs.microsoft.com/en-us/cpp/build/reference/subsystem-specify-subsystem
DQ2 is using `int main(int, char**)`, which is why the resulting link output is:
```
       3 subsystem (Windows CUI)
```
"main" in DolphinWX is actually defined here: https://github.com/dolphin-emu/dolphin/blob/master/Externals/wxWidgets3/include/wx/msw/init.h#L69 as `WinMain`, hence:
```
       2 subsystem (Windows GUI)
```
I tested replacing DQ2's current `main` with `int __stdcall WinMain(HINSTANCE hInstance, HINSTANCE hPrevInstance, char* lpCmdLine, int nCmdShow)` and confirmed the resulting binary is "GUI", and a console is not shown.
I think for now this should just be put on the backburner. As stated on IRC the console is still giving debug output which is better to see than hide at this point, and additionally this should be done _properly_, when it is changed. Meaning, unicode commandline args to DQ2 should behave as expected. At the moment this is not true because of cppoptparse (a regression introduced in DolphinWX as well).